database/migrations/2022_06_13_195413_add_permanent_to_user_account_history.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddPermanentToUserAccountHistory extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('osu_user_banhistory', function (Blueprint $table) {
+            $table->boolean('permanent')->default(0);
```

Comment:
could use period=0 to mean permanent, instead of a new column

Reply:
The question is: what goes in the `period` column when `permanent` is set to `1`?

Reply:
0 just like note? Thing is `note`s have period of 0 and they're not visible after `BAN_PERSIST_DAYS`... Unless we're changing it to be always visible as well.

Reply:
Aha, if there's already a standard for it then that's fine 👍